package com.numberfacts.domain.entities.exceptions

class UnexpectedErrorException(cause: Throwable): Exception(cause)
